mne/decoding/time_gen.py

```diff
@@ -359,7 +354,7 @@ def predict(self, epochs, test_times=None):
                                  for t_test in t_train]
                                  for t_train in test_times_['slices']]
         # Store all testing times parameters
-        self.test_times_ = test_times_
+        self.test_times = test_times_
```

Comment:
yes so when you call fit on new epochs you don't have any memory of the
first call to fit. clear?